src/Support/SupportData.php

```diff
@@ -826,7 +825,7 @@ protected function process_raw_post_errors( WP_Post $amp_validated_post ) {
 			$error_source_slugs = wp_list_pluck( $error_sources, 'error_source_slug' );
 			$error_source_slugs = array_values( array_unique( $error_source_slugs ) );
 
-			if ( ! empty( $error_source_slugs ) && is_array( $error_source_slugs ) ) {
+			if ( ! empty( $error_source_slugs ) ) {
```

Comment:
In PHP<8.0, `array_values()` can return `null` if it is not passed an array. Nevertheless, we have ensured above that the `$error_sources` input is an array.